content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,128 @@
+# To use, add an image named `featured.jpg/png` to your page's folder. 
+[image]
+  # Caption (optional)
+  caption = ""
+
+  # Focal point (optional)
+  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
+  #focal_point = "Center"
++++
+
+  
+
+Any open source project is successful only if the community keeps contributing regularly. We want to encourage our community members and 
+also anyone who wants to start contributing to KubeEdge. To sweeten the deal we have created a contest for you. To put it in simple words,
+
+{{% alert note %}}
+
+Contribute to KubeEdge and Win a ticket to **[KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019.](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019/)**
+
+{{% /alert %}}
+
+That's right! Contribute and win. Contribution does not mean only code contribution. You can contribute to KubeEdge in various ways.
+
+## **How to participate ?**
+
+{{% alert note %}}
+
+1. Raise pull request (PR) either for feature development / test code development (may be unit test code, edge module test code or end to end test code) in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+2. Identify defects, raise issues in respective repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+3. Re-solving existing issues in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+4. Share requirements by creating issues in repo [kubeedge](https://github.com/kubeedge/kubeedge).
+5. Writing blogs about KubeEdge either in the [KubeEdge website](https://kubeedge.io) (on PR approved & merged, this gets published in kubeedge.io website blog) or in other technical blogging site. Please refer here to know how to write a [KubeEdge blog](https://github.com/kubeedge/website/wiki/How-to-write-a-blog).
+Submit your blog details  [here](https://docs.google.com/forms/d/e/1FAIpQLSd9luGaHWt5jABEQjtr11CgWLVw7Px6tJf0TIjidbl3UL_Izg/viewform).
+6. Create your own sample applications and demo examples to illustrate possible use case(s) of using KubeEdge in repo [examples](https://github.com/kubeedge/examples).
+
+{{% /alert %}}
+
+## **Who can participate ?**
+
+At the moment most of the contributors are from Huawei organization. We want to encourage contributions from individual contributors or individuals of other organizations.  
+Contributions made from accounts linked to email ids of huawei organization will not be eligible for participation. 
+The contest is open to all other members.
+
+
+## **How the winners are selected ?**
+
```

Comment:
Contribution can be made in the following various ways. Please see below for contribution requirements and how we select winners.